libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_DNS.c

```diff
@@ -420,6 +420,7 @@ uint32_t FreeRTOS_gethostbyname_a( const char *pcHostName, FOnDNSEvent pCallback
 uint32_t ulIPAddress = 0UL;
 TickType_t xReadTimeOut_ms = ipconfigSOCK_DEFAULT_RECEIVE_BLOCK_TIME;
 TickType_t xIdentifier = 0;
+configASSERT(pcHostName);
```

Comment:
This needs to be a boolean expression.  Likewise for the `configASSERT(pcName)` that occurs later in this file.  It's pedantic, I know.

Reply:
I have added an `if` statement here instead of `configASSERT`.